### PR TITLE
Zig suboptimal 5s pause

### PR DESCRIPTION
**Pull request by ityonemo**
## Description
Updates Zig impl with suboptimal 5s pauses

## Contributing requirements
* [x] I read the contribution guidelines in CONTRIBUTING.md.
* [x] I placed my solution in the correct solution folder.
* [x] I added a README.md with the right badge(s).
* [x] I added a Dockerfile that builds and runs my solution.
* [x] I selected `drag-race` as the target branch.
* [x] All code herein is licensed compatible with BSD-3.

### CODE REVIEW

File: PrimeZig/solution_3/README.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Zig solution by ManDeJan and ityonemo and SpexGuy
 
+## WARNING
+The runners of the Primes program have instituted an limitation to the
```
Comment (rbergen) [inline PrimeZig/solution_3/README.md]: I think that in the GitHub context, the de-facto word for what you call "runners" is "maintainers". You can decide if you want to change this or not; I'm happy to merge either way.

**Comment by rbergen** [comment]
There is a build failure in a different solution (Zig/solution_1) that blocks the merge of this PR. I have opened #754 to address this. After its review and merge, you can sync your branch with this one, and we should be good to proceed with merging this.

**Comment by rbergen** [comment]
@ityonemo If you can please sync your PR branch with this one, then we should have CI working again. I will merge once that has been confirmed.

**Comment by rbergen** [comment]
I'm confused. This PR now includes files that belong to a host of other solutions (Clipper/solution_1, Crystal/solution_2, Mixed/solution_3, Rust/solution_1).

**Comment by ityonemo** [comment]
I have no idea what happened, I tried rebasing off of the drag-race branch.  This is what I got

**Comment by rbergen** [comment]
I'm sorry, but I can't merge this in this state. The diff can't include files in other solutions to which no changes should have been made.

**Comment by ityonemo** [comment]
Can you just roll it back to where it was at #750 and then merge in #754?  I don't know how to do that except by rebasing against develop, I tried everything in my personal git toolbox, unfortunately I am *just not that good at git*

**Comment by ityonemo** [comment]
I force-pushed it back to where it was before.  I have no idea how to get 754 in from this point.

**Comment by rbergen** [comment]
I understand the frustration, but I can't do anything about this directly. The reason is that in the end, a PR is a comparison between the base branch (this one) and the source branch (yours). 

That being as it is, I think we _can_ try a somewhat pragmatic approach. I've just opened a PR on your fork and source branch for this PR, that adds the #754 Dockerfile for Zig/solution_1 to your branch. If you merge that, we should be in a better place with this one, too.

**Comment by rbergen** [comment]
@ityonemo Thank you, it seems we're good to merge on this PR, now.

**Pull request merged**